pwem/objects/data.py

```diff
@@ -2296,6 +2296,8 @@ def classifyItems(self,
                 ref = newItem.getClassId()
                 if ref is None:
                     raise Exception('Particle classId is None!!!')
+                if ref == 0:
```

Comment:
I guess this means that None is an invalid class and raises an exception but 0 means that we need to skip the class and the particle, right?

Reply:
correct!